pkgs/development/python-modules/swspotify/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, python3 }:
+
+buildPythonPackage rec {
+  pname = "SwSpotify";
+  version = "1.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0mrrg3zcm2ppxxzcva8nbyj1dkcplif4a88mvapphfd91a1d521q";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    requests flask-cors dbus-python
+  ];
```

when in `python-modules` do not inherit from python3, as this will always be `python38` and you will have interpreter version issues.

instead, import each python package individually
```suggestion
  propagatedBuildInputs = [
    requests flask-cors dbus-python
  ];
```